Announce Docs SIG with SIG page and blog post (#2246)
jenkins-infra/jenkins.io: content/blog/2019/05/2019-05-11-docs-sig-announcement.adoc

---
:layout: post
:title: "Jenkins Documentation Special Interest Group"
:tags:
- documentation
- docs
- community
:author: markewaite
:sig: docs
---

image:/images/logos/256.png[Jenkins, role=center, float=right]

We're pleased to announce the formation of the Jenkins Documentation Special Interest Group.

> The Docs SIG will encourage, create, and review documentation contributions and improve documentation with contributors and external communities.

Reviewer: ```suggestion
The Docs SIG encourages contributors and external communities to create and review Jenkins documentation. 
```
See if this works for you.  The existing sentence is a bit obtuse and doesn't parse well so I fiddled with it.  You may be able to do better.

See the link:/sigs/docs[Special Interest Group Overview] for more details and plans.

== How can I help?

> The Jenkins Documentation SIG would love to have your help with:

Reviewer: ```suggestion
The Jenkins Documentation SIG would love to have you join us to help with:
```
The existing text sounds a bit like the Jenkins Documentation SIG is an annointed group that welcomes help from you peons, whereas I'm thinking that, by contributing, one becomes a part of the SIG.

* reviewing and fixing link:https://issues.jenkins-ci.org/issues/?jql=project%20%3D%20%22Jenkins%20Website%22%20AND%20status%20!%3D%20Done[open bug reports]
* reviewing link:https://github.com/jenkins-infra/jenkins.io/pulls[Jenkins documentation pull requests]
* reviewing link:https://github.com/jenkins-x/jx-docs/pulls[Jenkins X documentation pull requests]

== How can I fix a documentation bug?

Instructions for contributing to the Jenkins documentation are in the link:https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#getting-started[CONTRIBUTING file] of the site repository.
Follow the instructions in that file and submit pull requests for review.

Instructions for contributing to the Jenkins X documentation are on the link:https://jenkins-x.io/contribute/documentation/[Jenkins X documentation site].

> Follow those instructions and submit pull requests for review.

Reviewer: ```suggestion
Follow the instructions in that file and submit pull requests for review.
```
Maybe nice to use the same construction in both paragraphs?  I made this match the previous one but I'm happy with either construction.

== How can I evaluate a pull request?

Pull requests for the Jenkins project are reviewed in the link:https://github.com/jenkins-infra/jenkins.io/pulls[Jenkins documentation repository].

> Login to GitHub with your credentials and add your review comments to pull requests.

Reviewer: ```suggestion
Log in to GitHub with your credentials and add your review comments to pull requests.
```
